File: linearSearch/linearSearch.js
function linearSearch(arr, item) {
  // Go through all the elements of arr to look for item.
  for (var i = 0; i < arr.length; i++) {
    if (arr[i] === item) { // Found it!
      return i;
    }
  }
  // Item not found in the array.
  return null;
}

linearSearch([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20], 13);

File: linearSearch/linearSearch.py
def linearSearch(arr, item):
  for i in range(0, len(arr)):
    if (arr[i] == item):
      return i
  return -1

print("Element found at index "+str(linearSearch([1, 2, 3, 5, 7, 11, 13, 17, 23, 31], 13)))
